deploy/deployFIFOMarket.ts

```diff
@@ -0,0 +1,43 @@
+import { Logger, LogLevel } from '@ethersproject/logger';
+import { DeployFunction } from 'hardhat-deploy/types';
+import {
+  STAGING_NORI_FEE_WALLET_ADDRESS,
+  PROD_NORI_FEE_WALLET_ADDRESS,
+} from '../constants/addresses';
+import { deployFIFOMarketContract, finalizeDeployments } from '../utils/deploy';
+import { getCertificate } from '../utils/contracts';
+
+export const deploy: DeployFunction = async (env) => {
+  const hre = env as unknown as CustomHardHatRuntimeEnvironment;
+  Logger.setLogLevel(LogLevel.DEBUG);
+  hre.log(`deployFIFOMarket`);
+  const feeWallet =
```

Comment:
can we just make this an environment variable?

Reply:
Makes it more work for the person deploying, but I'm generally in favor of code not thinking too much about where it's running...  let's talk about this.